src/FSharpVSPowerTools.Core/InterfaceStubGenerator.fs

```diff
@@ -401,7 +401,7 @@ module InterfaceStubGenerator =
 
     let internal (|LongIdentPattern|_|) = function
         | SynPat.LongIdent(LongIdentWithDots(xs, _), _, _, _, _, _) ->
-            let (name, range) = xs |> Seq.map (fun x -> x.idText, x.idRange) |> Seq.last
+            let (name, range) = xs |> List.map (fun x -> x.idText, x.idRange) |> List.last
```

Comment:
as I see you ultimately want to get the last element of the list and then take its constituents. So why do you need to map the entire sequence if you are ignoring results of projection. To me it should be smth like
```fsharp
    let last = List.last xs
    Some (last.idText, last.idRange)
```